pkg/plugin/plugin_test.go

```diff
@@ -332,7 +328,7 @@ var _ = Describe("CNI Plugin", func() {
 				"type": "ovs",
 				"bridge": "%s",
 				"mtu": %d
-			}`, BRIDGE_NAME, MTU)
+			}`, bridgeName, MTU)
```

Comment:
if you rename constants to lower came case, we should rename MTU too. It should not be public.

Reply:
yes, renamed the constant.